perllib/FixMyStreet/App/Form/ManifestTheme.pm

```diff
@@ -0,0 +1,66 @@
+package FixMyStreet::App::Form::ManifestTheme;
+
+use Path::Tiny;
+use File::Copy;
+use Digest::SHA qw(sha1_hex);
+use File::Basename;
+use HTML::FormHandler::Moose;
+use FixMyStreet::App::Form::I18N;
+extends 'HTML::FormHandler::Model::DBIC';
+use namespace::autoclean;
+
+has 'cobrand' => ( isa => 'Str', is => 'ro' );
+
+has '+widget_name_space' => ( default => sub { ['FixMyStreet::App::Form::Widget'] } );
+has '+widget_tags' => ( default => sub { { wrapper_tag => 'p' } } );
+has '+item_class' => ( default => 'ManifestTheme' );
+has_field 'cobrand' => ( required => 0 );
```

Comment:
Could this be a dropdown, from `available_cobrand_classes` like ReportExtraFields does?

Reply:
That'd be nicer, yes.